packages/expo-cli/src/commands/publish-modify.ts

```diff
@@ -61,14 +61,22 @@ export default function (program: Command) {
     .helpGroup('publish')
     .option('--channel-id <channel-id>', 'This flag is deprecated.')
     .option('-c, --release-channel <channel-name>', 'The channel to rollback from. (Required)')
-    .option('-s, --sdk-version <version>', 'The sdk version to rollback. (Required)')
+    .option(
+      '-s, --sdk-version <version>',
+      'The sdk version to rollback. (Required if runtime version is not specified)'
```

Comment:
```suggestion
      'The SDK version to rollback. (Required if runtime version is not specified)'
```

(not introduced in this PR but we tend to capitalize SDK)